Client/game_sa/CEntitySA.cpp

```diff
@@ -45,7 +45,7 @@ CRect* CEntitySAInterface::GetBoundRect_(CRect* pRect)
     TransformFromObjectSpace(point, vecMax);
     rect.StretchToPoint(point.fX, point.fY);
     *pRect = rect;
-    pRect->FixIncorrectTopLeft();            // Fix custom map collision crashes in CPhysical class (infinite loop)
```

Comment:
Btw the comment indentation change here is probably not in line with the auto-formatter. Try Format Selection? (I think as a shorthand if you type a semicolon at Line 50 it will just format the entire method, and then you can remove the semicolon)

Reply:
I modified the indentation manually. Fixed now.